src/components/Contact/contact.css

```diff
@@ -0,0 +1,37 @@
+.contact-form {
+  margin: 0 auto;
+}
+
+.form-input,
+.form-textarea {
+  width: 100%;
+  box-sizing: border-box;
+  border-radius: 20px;
+  padding: 10px 20px;
+  font-size: 1em;
+  font-family: inherit;
+  background-color: var(--primary-element-color);
+  margin-bottom: 30px;
```

Comment:
don't add external rules for components. Seems like `.form-input` and `.form-textarea` are common classes for such elements. If you add new one - the margin can be not needed. And after many such happening there tons of redundant code.
Better apply wrapper for such separations.

Reply:
Can explain better in live chat

Reply:
fixed